# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1696891576 -7200
# Parent  c02611154879d8b63f85b8eaa2efe4c644ae8afc
Misc: Allow for identity without outgoing server (for example movemail).

diff --git a/mail/components/about-support/content/accounts.js b/mail/components/about-support/content/accounts.js
--- a/mail/components/about-support/content/accounts.js
+++ b/mail/components/about-support/content/accounts.js
@@ -62,16 +62,17 @@ var AboutSupport = {
    */
   _getSMTPDetails(aAccount) {
     let defaultIdentity = aAccount.defaultIdentity;
     let smtpDetails = [];
 
     for (let identity of aAccount.identities) {
       let isDefault = identity == defaultIdentity;
       let smtpServer = MailServices.smtp.getServerByIdentity(identity);
+      if (!smtpServer) continue;
       smtpDetails.push({
         identityName: `${identity.identityName} (${identity.key})`,
         name: `${smtpServer.displayname} (${smtpServer.key})`,
         authMethod: smtpServer.authMethod,
         socketType: smtpServer.socketType,
         isDefault,
       });
     }
